microservice/src/test/java/de/gsi/serializer/helper/CmwLightHelper.java

```diff
@@ -388,7 +391,11 @@ public static void testPerformancePojo(final int iterations, final TestDataClass
 
             byteBuffer.reset();
 
-            ioSerialiser.deserialiseObject(outputObject);
+            final Object returnedObject = ioSerialiser.deserialiseObject(outputObject);
+
+            if (outputObject != returnedObject) {
```

Comment:
Codacy found an issue: [Use equals() to compare object references.](https://app.codacy.com/gh/GSI-CS-CO/chart-fx/pullRequest?prid=6382991)